voice/text-to-speech-outbound-slim/index.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use \Psr\Http\Message\ServerRequestInterface as Request;
+use \Psr\Http\Message\ResponseInterface as Response;
+use Slim\Factory\AppFactory;
+require __DIR__ . '/vendor/autoload.php';
+
+
+$app = AppFactory::create();
+
+$app->get('/webhook/answer', function (Request $request, Response $response) {
+    $client = new GuzzleHttp\Client();
+    $apiResponse = json_decode($client->get('http://api.icndb.com/jokes/random?limitTo=[nerdy]')->getBody());
+
+    $data = [
+        [
+            'action' => 'talk',
+            'voiceName' => 'Amy',
+            'text' => $apiResponse->value->joke
+        ]
+    ];
+
+    $payload = json_encode($data);
```

Comment:
I'm not sure the added temporary variable is needed here. Could simply json_encode() inside the write() call. Not a major issue. (I hate temp vars.)

Reply:
Yes, I've removed the unnecessary temp variable.